definitions/divorce/json/CaseEvent/CaseEvent-deemed-and-dispensed-nonprod.json

```diff
@@ -141,5 +141,220 @@
+    "PostConditionState": "ServiceApplicationNotApproved",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "16/09/2020",
+    "CaseTypeID": "DIVORCE",
+    "ID": "IssueAOSOffline_Respondent_FromServiceAppRejected",
+    "Name": "Issue AOS pack to respondent",
+    "Description": "Answer received",
+    "DisplayOrder": 9,
+    "PreConditionState(s)": "ServiceApplicationNotApproved",
+    "PostConditionState": "AosAwaiting",
+    "RetriesTimeoutURLAboutToSubmitEvent": "120,120",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "16/09/2020",
+    "CaseTypeID": "DIVORCE",
+    "ID": "IssueAOSOffline_CoRespondent_FromServiceAppRejected",
+    "Name": "Issue AOS pack - Co-respondent",
+    "Description": "Answer received",
+    "DisplayOrder": 10,
+    "PreConditionState(s)": "ServiceApplicationNotApproved",
+    "PostConditionState": "ServiceApplicationNotApproved",
+    "SecurityClassification": "Public",
+    "ShowSummary": "Y"
+  },
+  {
+    "LiveFrom": "16/09/2020",
+    "CaseTypeID": "DIVORCE",
+    "ID": "reissueFromServiceAppRejected",
+    "Name": "Reissue petition",
+    "Description": "Reissue petition",
+    "DisplayOrder": 11,
+    "PreConditionState(s)": "ServiceApplicationNotApproved",
+    "PostConditionState": "AwaitingReissue",
+    "SecurityClassification": "Public",
+    "ShowSummary": "Y"
+  },
+  {
+    "LiveFrom": "16/09/2020",
+    "CaseTypeID": "DIVORCE",
+    "ID": "transferCaseFromServiceAppRejected",
+    "Name": "Transfer from CTSC to RDC",
+    "Description": "Transfer from CTSC to RDC",
+    "DisplayOrder": 12,
+    "PreConditionState(s)": "ServiceApplicationNotApproved",
+    "PostConditionState": "Issued",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "18/09/2020",
+    "CaseTypeID": "DIVORCE",
+    "ID": "startAosFromServiceAppRejected",
+    "Name": "AOS started",
+    "Description": "AOS started",
+    "DisplayOrder": 1,
+    "PreConditionState(s)": "ServiceApplicationNotApproved",
+    "PostConditionState": "AosStarted",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "16/09/2020",
+    "CaseTypeID": "DIVORCE",
+    "ID": "aosNotReceivedFromServiceAppRejected",
+    "Name": "AOS Not Received within SLA",
+    "Description": "AOS Not Received within SLA",
+    "DisplayOrder": 4,
+    "PreConditionState(s)": "ServiceApplicationNotApproved",
+    "PostConditionState": "AosOverdue",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "16/09/2020",
+    "CaseTypeID": "DIVORCE",
+    "ID": "solAosNotReceivedFromServiceAppRejected",
+    "Name": "AOS Not Received within SLA",
+    "Description": "AOS Not Received within SLA",
+    "DisplayOrder": 5,
+    "PreConditionState(s)": "ServiceApplicationNotApproved",
+    "PostConditionState": "AosOverdue",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "16/09/2020",
+    "CaseTypeID": "DIVORCE",
+    "ID": "startAosFromServiceAppRejected",
```

Comment:
As mentioned above, you can remove this one as it already exists in the same file and also the authorisations you had added for it :)